Ginger/Ginger/Activities/ActionsPOMListViewHelper.cs

```diff
@@ -0,0 +1,364 @@
+{
+    public class ActionsPOMListViewHelper : IListViewHelper
+    {
+        Act mAct;
+        public Context mContext;
+        private Agent mAgent;
+        public General.eRIPageViewMode PageViewMode { get; set; }
+
+        UcListView mListView = null;
+        public UcListView ListView
+        {
+            get
+            {
+                return mListView;
+            }
+            set
+            {
+                if (mListView != value)
+                {
+                    //if (mListView != null)
+                    //{
+                    //    mListView.UcListViewEvent -= ListView_UcListViewEvent;
+                    //}
+                    mListView = value;
+                    //if (mListView != null)
+                    //{
+                    //    mListView.UcListViewEvent += ListView_UcListViewEvent;
+                    //}
+                }
+            }
+        }
+
+        public bool AllowExpandItems { get; set; } = true;
+
+        public bool ExpandItemOnLoad { get; set; } = false;
+
+        public ActionsPOMListViewHelper(Context context, General.eRIPageViewMode pageViewMode)
+        {
+            mContext = context;
+            PageViewMode = pageViewMode;
+        }
+
+        public void SetItem(object item)
+        {
+            if (item is ApplicationPOMModel)
+            {
+                mAct = (Act)item;
+            }
+            else if (item is ucButton)
+            {
+                mAct = (Act)(((ucButton)item).Tag);
+            }
+        }
+
+        public string GetItemNameField()
+        {
+            return nameof(ElementInfo.ElementName);
+        }
+
+        public string GetItemMandatoryField()
+        {
+            return null;
+        }
+
+        public string GetItemDescriptionField()
+        {
+            return nameof(ElementInfo.ElementTypeEnum);
+        }
+
+        public string GetItemErrorField()
+        {
+            return null;
+        }
+
+        public string GetItemNameExtentionField()
+        {
+            return null;
+        }
+
+        public string GetItemTagsField()
+        {
+            return null; 
+        }
+
+        public string GetItemExecutionStatusField()
+        {
+            return null;
+        }
+
+        public string GetItemActiveField()
+        {
+            return null;
+        }
+
+        public ListItemUniqueIdentifier GetItemUniqueIdentifier(object item)
+        {
+            SetItem(item);
+            //return new ListItemUniqueIdentifier() { Color = mAct.GroupColor, Tooltip = mAct.Name };
+            return null;
+        }
+
+        public string GetItemIconField()
+        {
+            return nameof(ElementInfo.ElementTypeImage);
+        }
+
+        public string GetItemIconTooltipField()
+        {
+            return nameof(Activity.ActivityType);
+        }
+
+        public List<ListItemOperation> GetListOperations()
+        {
```

Comment:
I need to see how POM actions list looks in existing solution and how in new to make sure we didn't missed anything